meta-client/src/main/java/org/apache/helix/metaclient/recipes/lock/LockInfo.java

```diff
@@ -171,23 +169,23 @@ public void setClientData(String clientData) {
    * Get the value for GRANTED_AT attribute of the lock
    * @param grantTime Long representing the time at which the lock was granted
    */
-  public void setGrantedAt(Long grantTime) {
+  private void setGrantedAt(Long grantTime) {
```

Comment:
where is this func called?

Reply:
Helper function when creating LockInfo's